### PR TITLE
save edit account button and fixed signup button

### PR DESCRIPTION
**Pull request by Sihellman**

### CODE REVIEW

File: client/src/App.js [hunk 1]
```diff
@@ -1,26 +1,40 @@
...
 const App = () => {
     const [user, setUser] = useState('');
+
     const [toggleLogInSignUp, setToggleLogInSignUp] = useState(true);
+    useEffect(() => {
+        const loggedInUser = JSON.parse(localStorage.getItem('user'));
+        if (loggedInUser) {
+            setUser(loggedInUser);
+        }
+    }, []);
```
Comment (Yatkins) [inline client/src/App.js:17]: Just curious about the purpose of the square brackets at the end, looks great!

File: client/src/components/Head.js [hunk 2]
```diff
@@ -16,17 +16,29 @@ import {
     NavbarText
 } from 'reactstrap';
 import {Link} from "react-router-dom";
+import {useHistory} from "react-router-dom";
```
Comment (Yatkins) [inline client/src/components/Head.js:19]: how does useHistory work?

Comment (Yatkins) [reply client/src/components/Head.js:19]: and useEffect?

**Review by Yatkins — APPROVED** [review]
Looks really good! Lots of work and new methods like spread operator that I haven't seen before.

**Comment by Sihellman** [comment]
thank you!!!!!!!!!!! 
I got from here:
https://www.freecodecamp.org/news/how-to-persist-a-logged-in-user-in-react/
I'm not sure about the square brackets either. 
UseHistory allows you to store the history of the pages you visit so that you can push the back button in the browser and go to the last page visited.
thank you so much for noticing!!!

**Pull request merged**